cli.js

```diff
@@ -88,7 +89,8 @@ var cli = meow([
 		'require',
 		'timeout',
 		'source',
-		'match'
+		'match',
+		'pool-size'
```

Comment:
Certainly. I was going off fabfile.org's as it was closest to my experiences. Ill change in the next push